mkdocs.yml

```diff
@@ -0,0 +1,49 @@
+site_name: Perseo Context-aware Complex Event Processing
+site_description: Perseo Documentation
+repo_url: https://github.com/Ficodes/perseo-fe
+theme: readthedocs
+extra_css: ['https://fiware.org/style/fiware_readthedocs.css', "https://fiware.org/style/fiware_readthedocs_core.css"]
+#google_analytics: ['UA-84269043-1', 'wirecloud.readthedocs.io']
+
+pages:
+- 'Introduction': 'index.md'
+- 'Architecture': 'architecture/architecture.md'
+- 'Admin Guide': 'admin/index.md'
+- 'User Guide': 'user/index.md'
+- 'O&M':
+    - 'Deployment': 'admin/deployment.md'
+    - 'Configuration': 'admin/old_configuration.md'
+    - 'Administration': 'admin/admin.md'
+    - 'Metrics API': 'API/metrics_api.md'
+    - 'Logs & Alarms': 'admin/logs.md'
+- 'Open API':
+    - 'API': 'API/api.md'
+    - 'API overview': 'API/index.md'
+    - 'Actions': 'API/pep_actions.md'
+    - 'Plain rules': 'API/plain_rules.md'
+    - 'Errors': 'API/errors.md'
+    - 'Events': 'API/event_api.md'
+- 'Data model': 'models/models.md'
+#- 'Available actions for PEP': 'API/pep_actions.md'
+- 'Documentation for developers of perseo': 'developer/development.md'
+- 'Roadmap': 'roadmap.md'
+
+markdown_extensions:
+#    - markdown_github.headerid
+    - markdown.extensions.tables
+    - pymdownx.betterem
+    - pymdownx.tilde:
+        subscript: False
+    - pymdownx.superfences
+    - pymdownx.magiclink:
+        repo_url_shortener: True
+        repo_url_shorthand: True
+        provider: "github"
+        user: "Ficodes"
+        repo: "perseo-fe"
+    - pymdownx.tasklist
+    - codehilite:
```

Comment:
Codehilite and Superfences have conflicts with FIWARE CSS - see https://github.com/Wirecloud/wirecloud/pull/389

Reply:
ok. Fixed in ef4582b